pkg/server/policyrules.go

```diff
@@ -26,6 +26,8 @@ import (
 	multiv1beta1 "github.com/k8snetworkplumbingwg/multi-networkpolicy/pkg/apis/k8s.cni.cncf.io/v1beta1"
 
 	//v1 "k8s.io/api/core/v1"
```

Comment:
Could you please remove this comment outed line if you don't mind?

Reply:
Yes, of course! My fault with the IDE autoimport feature